src/hooks/usePriceHistory.ts

```
@@ -48,34 +55,36 @@ export function usePriceHistory(
         windowStart = now - 7 * 24 * 60 * 60 * 1000;
         break;
-      case "30d":
+      case '30d':
         hours = 30 * 24;
         windowStart = now - 30 * 24 * 60 * 60 * 1000;
         break;
-      case "90d":
-        windowStart = sortedData[0]?.timestamp || now;
-        now = sortedData[sortedData.length - 1]?.timestamp || now;
+      case '90d':
+        const firstPoint = safeArrayFirst(sortedData);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)